velox/substrait/SubstraitToVeloxPlanValidator.cpp

```diff
@@ -18,8 +18,30 @@
 #include "TypeUtils.h"
 #include "velox/expression/SignatureBinder.h"
 
+#include <google/protobuf/wrappers.pb.h>
+
 namespace facebook::velox::substrait {
 
+/// @brief Return whether a config is set as true in AdvancedExtension
+/// optimization.
+/// @param extension Substrait advanced extension.
+/// @param config the key string of a config.
+/// @return Whether the config is set as true.
+bool configSetInOptimization(
```

Comment:
How about moving this function into [SubstraitParser](https://github.com/oap-project/velox/blob/bd4d488269659c205536475fa2487327611a7f26/velox/substrait/SubstraitParser.cpp) so we can reuse here and there?